stream-chat-android-ui-components/src/main/res/values/attrs_message_composer_view.xml

```diff
@@ -151,6 +151,8 @@
 
         <attr name="streamUiMessageComposerCooldownTimerBackgroundDrawable" format="reference" />
 
+        <attr name="streamUiMessageComposerShowMessageReplyView" format="boolean" />
```

Comment:
Could we leave an XML description here?
I know we don't have a habit of doing it, but it's useful to users.